stdlib/public/SDK/Foundation/JSONEncoder.swift

```diff
@@ -300,45 +331,141 @@ fileprivate struct _JSONKeyedEncodingContainer<K : CodingKey> : KeyedEncodingCon
         self.container = container
     }
 
+    // MARK: - Coding Path Operations
+
+    /// Performs the given closure with the given key pushed onto the end of the current coding path.
+    ///
+    /// - parameter key: The key to push. May be nil for unkeyed containers.
+    /// - parameter work: The work to perform with the key in the path.
+    fileprivate mutating func with<T>(pushedKey key: CodingKey, _ work: () throws -> T) rethrows -> T {
```

Comment:
I think you'll need to rebase your changes on the latest `master`. I got rid of `with<T>` a while back and we don't use it anymore, so this is adding it back.

Reply:
I see, I guess I just brought it back from the dead here.